Modules/Filtering/Smoothing/include/itkFFTDiscreteGaussianImageFilter.h

```diff
@@ -151,7 +155,12 @@ class ITK_TEMPLATE_EXPORT FFTDiscreteGaussianImageFilter : public DiscreteGaussi
   itkGetConstMacro(KernelSource, FFTDiscreteGaussianImageFilterEnums::KernelSource);
 
 protected:
-  FFTDiscreteGaussianImageFilter() = default;
+  FFTDiscreteGaussianImageFilter()
+  {
```

Comment:
Do we need the body? I think both of these assignments can be made at the point of declaration.

Reply:
At a glance I can't find any other examples of initializing filter members at the point of declaration, it seems to be standard practice to initialize in the constructor. Without distinct benefit I'd prefer to stick to the standard approach.

Reply:
It was [not possible before C++11](https://stackoverflow.com/questions/24149924/has-the-new-c11-member-initialization-feature-at-declaration-made-initializati), and not all classes have been converted to the new syntax. But initialization at declaration is the preferred style, and should be used in new code.

Reply:
👍 Thanks for the link and background, it seems like there is benefit especially in the context of overridden constructors. I will make the change.